.github/actions/setup-deps/action.yaml

```diff
@@ -123,6 +125,8 @@ runs:
         CONDA_OPT_DEPS: |
           ${{ inputs.chemfiles-python }}
           ${{ inputs.clustalw }}
+          ${{ inputs.dask }}
+          ${{ inputs.dask-distributed }}
```

Comment:
As discussed in https://github.com/MDAnalysis/mdanalysis/issues/4158#issuecomment-1714302736, `dask-distributed` should be removed (distributed memory parallelism will be explored and provided via an MDAKit).

Reply:
Fixed.